### ISSUE TITLE
Failed to load

### ISSUE THREAD

**Issue by glea94** [CGessinger/Creatures-And-Beasts-Minecraft-Mod#41]
Works fine yesterday but today i got the issue for no explainable reason
https://pastebin.com/hVZp6n3H

**Comment by Dariensg** [comment]
The config has duplicated itself. Just go into the config file in /.minecraft/configs/ and open the Creatures and Beasts config. Then just delete the duplicate text and it should work as normal. This is a known bug, but it's unclear what's causing it.

**Comment by SwanX1** [comment]
[infernalstudios/Blazing-Config@3.0.3](https://github.com/infernalstudios/Blazing-Config/compare/3.0.2...3.0.3) has fixed it, just waiting on the config library version for this mod to be updated!
Note: The update won't fix existing configs, it will just fix the issue of it duplicating.

**Comment by Dariensg** [comment]
The config will no longer duplicate itself in version 1.5.0. Update the mod, delete the duplicate text in the config, and it shouldn't happen again.
